rust/src/lib.rs

```diff
@@ -15,7 +15,6 @@
  * 02110-1301, USA.
  */
 
-#![allow(ellipsis_inclusive_range_patterns)] // TODO: Remove when MSRV is higher than 1.24
```

Comment:
I think it might be better to add
```
#![allow(unknown_lints)]
```

With the change here, and Rust strict mode we get a warning in 1.24.1.. Without it, we get a strict error in 1.47.  I think its safe to jus allow unknown lints in the older code.

Reply:
My bad, I should have checked with the current compiler as well. I'll make the change right away.